### PR TITLE
list-ops: Add generator

### LINKED ISSUE #413: Add List Ops Test Generator
This issue is part of an overall initiative to complete #195.

The documentation on how to add generators can be found [here](https://github.com/exercism/csharp/blob/master/docs/GENERATORS.md). If you get stuck or have any questions about adding this generator, please do not hesitate to reach out!

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes #413

### CODE REVIEW

File: exercises/list-ops/ListOps.cs [hunk 1]
```diff
@@ -14,27 +14,27 @@ public static List<T> Reverse<T>(List<T> input)
...
     }
 
-    public static TOut Foldr<TIn, TOut>(Func<TIn, TOut, TOut> func, TOut start, List<TIn> input)
+    public static TOut Foldr<TIn, TOut>(List<TIn> input, TOut start, Func<TIn, TOut, TOut> func)
     {
         throw new NotImplementedException("You need to implement this function.");
     }
 
-    public static List<T> Concat<T>(List<List<T>> input)
+    public static List<object> Concat(List<object> input)
```
Comment (robkeim) [inline exercises/list-ops/ListOps.cs]: Why change this to a `List<object>` as opposed to the more generic `List<T>`?

Comment (ErikSchierboom) [reply exercises/list-ops/ListOps.cs]: Ah, that is a mistake. I'll correct it soon. Thanks for reviewing!

p.s. as you've noticed, I've done a lot of these generators the last couple of weeks. I requested a review on one of the first PR's, but having had no response for a couple of days, I figured you were either terribly busy or perhaps on holiday, which is why I've merged them myself. If you find anything not to your liking, please let me know!

p.p.s. Once all generators have been created, I'll be doing a refactoring of the generators code to hopefully simplify things.

Comment (ErikSchierboom) [reply exercises/list-ops/ListOps.cs]: @robkeim Fixed!

**Comment by robkeim** [comment]
Thanks @ErikSchierboom! Regarding your P.S. I was doing a bit of traveling and fell behind on my personal email (including reviewing your PRs). I'll have a look through them and let you know if I see anything that doesn't make sense. Looking forward to the P.P.S. :)

**Pull request merged**